monkey/common/agent_plugins/agent_plugin_type.py

```diff
@@ -2,7 +2,7 @@
 
 
 class AgentPluginType(Enum):
-    CREDENTIAL_COLLECTOR = "CredentialCollector"
+    CREDENTIAL_COLLECTOR = "Credential_Collector"
```

Comment:
`Credential Collector`

Also, should this be plural?

Reply:
No, it shouldn't be plural. According to how plugins are parsed right now, this needs to be the same as what the plugin tar name contains (in lowercase) AND what the plugin manifest has in the `plugin_type` field (which we've discussed is not good design and needs to be changed).

For example, for Mimikatz, the tar name needs to be `mimikatz-credentialcollector.tar` and the manifest needs to contain `plugin_type: CredentialCollector`.

I suggest we leave this as `CredentialCollector`. Adding a space would require the tar name to have a space (`mimikatz-credential collector.tar`), and adding an underscore would require the manifest to have `plugin_type: Credential_Collector`. Unless we decide to change plugin parsing before we get to migrating the credential collectors.

Reply:
No, it would work if you left this as is but changed the tar name to `MockCollector-credentialcollector.tar` instead of `MockCollector-credential_collector.tar`.

Reply:
We _have_ to go with `Credential_Collector` as the value in that case.

Reply:
Ok, that's fine. But should we rename the enum name and value to be plural? The plugin collects credential, not a single credential.

Reply:
@cakekoa you should rebase on develop and drop this commit. It is already done there.